config/services.yaml

```diff
@@ -53,6 +53,5 @@ services:
     OpenConext\ProfileBundle\Twig\I18n:
         tags: ['twig.extension']
 
-#    Surfnet\StepupSelfService\SamlStepupProviderBundle\Session\SessionFactoryWithAttributeBag:
-#        decorates: session.factory
-#        arguments: [ '@.inner' ]
+    OpenConext\EngineBlockApiClientBundle\:
+        resource: '../src/OpenConext/EngineBlockApiClientBundle'
```

Comment:
Do you need this. Looks like a noop in my eyes which can be calculated by the autoloader/autoconfigurator?